File: app/src/main/java/com/kunkunapp/allvideodowloader/webservice/Result.java
package com.kunkunapp.allvideodowloader.webservice;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {
    @SerializedName("key")
    @Expose
    private String key;
    @SerializedName("mrk")
    @Expose
    private Integer mrk;

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public Integer getMrk() {
        return mrk;
    }

    public void setMrk(Integer mrk) {
        this.mrk = mrk;
    }

}

File: app/src/main/java/com/cyberIyke/allvideodowloader/webservice/Result.kt
package com.cyberIyke.allvideodowloader.webservice

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {
    @SerializedName("key")
    @Expose
    var key: String? = null

    @SerializedName("mrk")
    @Expose
    var mrk: Int? = null
}